src/l1-resources/search-index/index.ts

```diff
@@ -244,21 +260,21 @@ export interface ApiAtlasFtsSynonymMappingDefinitionView {
    *
    * @schema ApiAtlasFTSSynonymMappingDefinitionView#Analyzer
    */
-  readonly analyzer?: string;
+  readonly analyzer: string;
 
   /**
    * Human-readable label that identifies the synonym definition. Each **synonym.name** must be unique within the same index definition.
    *
    * @schema ApiAtlasFTSSynonymMappingDefinitionView#Name
    */
-  readonly name?: string;
+  readonly name: string;
 
   /**
    * Data set that stores the mapping one or more words map to one or more synonyms of those words.
    *
    * @schema ApiAtlasFTSSynonymMappingDefinitionView#Source
    */
-  readonly source?: SynonymSource;
+  readonly source: SynonymSource;
```

Comment:
the change of certain fields (`analyzer`, `name`, `source`) to required was part of the latest changes when adding support for vector search?

Reply:
the changes are not only because vector support but also the 2 PRs by Adel to fix issues